core/templates/dev/head/css/oppia.css

```diff
@@ -574,6 +574,17 @@ textarea {
 .oppia-navbar-dropdown > li > a {
   color: #009688;
 }
+
+/* smooth color chang to the navbar-tabs */
+.oppia-navbar-tabs li a {
+
+  -webkit-transition: all .5s ease;
```

Comment:
alphabetize the properties, see our style [guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#css)

Reply:
ok. got it.